src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -24,6 +24,11 @@
     <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">6.0.2.0</AssemblyVersion>
     <PackageValidationBaselineVersion>6.1.0</PackageValidationBaselineVersion>
   </PropertyGroup>
+  
+  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp' and '$(IsPlaceholderTargetFramework)' != 'true'">
+    <ExtraMacros>#define netcoreapp</ExtraMacros>
+    <CoreAssembly>System.Runtime</CoreAssembly>
+  </PropertyGroup>
```

Comment:
Without these properties I would get the error:
```
C:\Users\calope\source\repos\maintenance-packages\artifacts\obj\System.Runtime.CompilerServices.Unsafe\Release\net6.0\version\version.h(9): error : syntax error at token '{' in: { [C:\Users\calope\source\repos\mainte
nance-packages\src\System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj::TargetFramework=net6.0]
```

because the autogenerated version.h file would be missing a bunch of values.